cookbook/power_meter.rst

```diff
@@ -24,6 +24,18 @@ And... that should already be it :)
     :align: center
     :width: 80.0%
 
+.. note::
+
+    Some energy meters have an exposed S0 port (which essentially just is a switch that closes), if 
+    that is the case the photo diode can be replaced with the following connection.
```

Comment:
```suggestion
    Some energy meters have an exposed S0 port (which essentially just is a switch that closes), if
    that is the case the photodiode can be replaced with the following connection.
```

Reply:
Additionally this kinda unrelated to the PR presented... Should this have been a separate PR?

Reply:
Well it could but this is more of an "update doc" pr so perhaps we could just update the name.